### DISCUSSION TITLE
Tilbageværende vagter på et event skal tælles fra dd. og frem

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#16] [Bugs] [answered]
På portalen opsummeres det, hvor mange vagter, der er tilbage til et event. F.eks. Cph Opera Festival har her 17 vagter tilbage. Screenshot er taget 28. august:

![image](https://github.com/user-attachments/assets/a22711b1-869f-4b4c-95d4-763283f41870)

![image](https://github.com/user-attachments/assets/138f37b5-c276-4ca2-802b-3434fa10fd0a)

Men går man ind på evenet, så er der 0 ledige vagter. 

Kigger vi i backend, så er summen af ledige vagter i fremtiden og i fortiden netop 17. 

Når det ifm. events opsummeres, hvor mange ledige vagter, der er tilbage, så skal den kun tælle fremtidige ledige vagter fra og med dd. med. Og kun vagter, hvor tilmelding er åben.

**Comment by antonlund18** [comment, marked as answer]
Oprettet som issue: https://github.com/xooom-community/product/issues/314
